src/node_messaging.cc

```diff
@@ -1273,6 +1273,99 @@ Maybe<bool> JSTransferable::Data::FinalizeTransferWrite(
   return ret;
 }
 
+std::shared_ptr<SiblingGroup> SiblingGroup::Get(const std::string& name) {
+  Mutex::ScopedLock lock(SiblingGroup::groups_mutex_);
+  if (name.empty()) return {};
```

Comment:
```suggestion
  if (name.empty()) return {};
  Mutex::ScopedLock lock(SiblingGroup::groups_mutex_);
```

(Not sure if it matters here a lot, but skipping unnecessary locks is a good practice because it can avoid unnecessary context switches if the mutex does happen to be locked already)

Reply:
Yep, that works.

Reply:
Done. I also went ahead and added ref/unref on the JavaScript `BroadcastChannel`. Those aren't standard but it matches what we've done with `MessagePort` so I think it makes sense.